src/components/InfoPanel.component.js

```diff
@@ -137,6 +171,25 @@ class InfoPanel extends Component {
           </div>
         )}
 
+        {!!media.length &&
+          isEditing &&
+          isRealTimePOI && (
+            <FieldGroup
+              inputType="text"
+              value={captions}
+              placeholder="Change Media Caption Here"
+              onChange={captions =>
+                modifyPOIFormCaption(
+                  this.props.carouselIndex,
+                  captions.target.value
+                )}
+            />
+          )}
+
+        {!!media.length && (
```

Comment:
this shouldn't be displayed if `isRealTimePOI` is true

Reply:
But I thought the purpose was to only be able to edit the carousel when we're editing the POI directly (as of now, this is the case; you can edit the caption when the left side of the screen shows the POI Form)